tests/resources/module/demo/vendor/composer/LICENSE

```diff
@@ -0,0 +1,21 @@
+
+Copyright (c) Nils Adermann, Jordi Boggiano
```

Comment:
Nils Adermann? Jordi Boggiano?

Reply:
Yes, it's Composer License: do you ask me to steal Composer work?

Reply:
Can you add a README under demo module just to explain why they are these files :D

Reply:
are you kidding? xD it's ``vendor`` folder, there is nothing special with this module.

What do you suggest as README content?

Reply:
Just say something like "Demo module, files automatically generated by composer", a simple sentence for guys don't know tests / composer / vendor directories :)

Reply:
Ok, fair enough :)

Reply:
fixed btw